crypto/fipsmodule/ec/p384.c

```diff
@@ -78,20 +95,20 @@ static inline uint8_t use_s2n_bignum(void) { return 1; }
 
 // The following four functions need bmi2 and adx support.
 #define p384_felem_mul(out, in0, in1) \
-  if (use_s2n_bignum()) bignum_montmul_p384(out, in0, in1); \
-  else fiat_p384_mul(out, in0, in1);
+  if (use_s2n_bignum_alt()) bignum_montmul_p384_alt(out, in0, in1); \
```

Comment:
I hope the cpu is good at branch prediction.

Reply:
I recall, comment was mostly a jinx; when we hit the random CPU with bad branch predictor.